scout/parse/variant/coordinates.py

```diff
@@ -80,9 +71,8 @@ def sv_end(pos, alt, svend=None, svlen=None):
         if match:
             end = int(match.group(2))
 
-    if svend == pos:
-        if svlen:
-            end = pos + svlen
+    if end is None and svlen:
```

Comment:
@dnil I made the code safer. I think it's fine to merge now and test before the release, but I can load a real case if you want!

Reply:
I see that you have no objections, merging! 😄

Reply:
Assuming you ran the tests and saw that all weird notations were fine, definitely. 😁

Reply:
The new version is indeed safer, but it also slightly changes the reading, in particular on the priority of using the matched-from-alt-str vs using calculated from pos and spleen. But, if we assume bug free callers, this should be a non-issue.

Reply:
I tested it with the VCF provided in the issue and it still worked. If you have a SV without an end  it will crash. This latter version checks for that- We can't foresee future bugs in the callers, but perhaps it's problems better solved in other repos, no?

Reply:
Sure, needed. While subtly changing the way we prioritise ends in a call with both alt len and svlen, but no end. I don't see it being a big issue, but if I've learned one thing about bioinformatics data, it's that if you have enough of it you usually encounter the weird exceptions. We should add at least small sample of current SV calls to the demo or test cases, but that can be another issue!

Reply:
And we will test before releasing so no worries.